src/applications/disability-benefits/all-claims/pages/privateMedicalRecordsRelease.js

```diff
@@ -63,6 +66,22 @@ export const uiSchema = {
         },
         'ui:required': formData => isCompletingForm0781(formData),
       },
+      treatedDisabilityNames: {
+        'ui:title': 'What conditions were you treated for?',
```

Comment:
I realize this probably was provided by Content, but I think "Which", instead of "What", would be a more accurate word choice as the question is being asked in the context of a multiple-choice list, not free-text input.